src/Libraries/RevitNodesUI/RevitNodesUI.csproj

```diff
@@ -156,7 +156,7 @@
   <ItemGroup>
     <EmbeddedResource Include="Properties\Resources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+      <LastGenOutput>Resources1.Designer.cs</LastGenOutput>
```

Comment:
oh my god, my bad. My mouse glitched up and instead of double click to open i dragged the resource file creating a copy of it. I deleted that thing as soon as I tried to build and ran into all the errors realizing what i did. I will try and build again so that gets overridden.